manifests/charts/base/files/gen-istio-cluster.yaml

```diff
@@ -6026,6 +6026,18 @@ spec:
     storage: true
 ---
 
+---
+# Source: base/templates/reader-serviceaccount.yaml
+# This service account aggregates the permissions for the revisions in a given cluster
+# Should be used for remote secret creation.
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: istio-reader-aggregate
```

Comment:
should we use the original name? like istio-reader-service-account or whatever it was? then folks don't need to change as much (?)

Reply:
Good point, changed back to original name